flybywiresim/aircraft
# Fix/a380x missing tooltip declarations

Fixes #9214

## Summary of Changes

I corrected the missing tooltip declarations in the LOK file, updated the autobrake tooltips key to match the LOK file, and created a new tooltip for the button that switches from true heading to magnetic. Previously, it was incorrectly using the "AP ENGAGE" tooltip.

## Screenshots (if necessary)



## References





## Additional context

The autobrake tooltips, except for RTO, were referencing a mismatched key in the LOK file.
Tooltips for the AUTO signs were undefined.
I overwrote the ENG 1 and ENG 2 de-ice tooltips because they were labeled as left and right engine de-ice.

Discord username (if different from GitHub):
lelouchtwt

## Testing instructions

The updates I made are purely visual. You only need to hover over the buttons to see the tooltips. There are different tooltips for each state.


## How to download the PR for QA

Every new commit to this PR will cause new A380X artifacts to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, find and click on the **PR Build** tab
1. Click on either **flybywire-aircraft-a380-842 (4K)** or **flybywire-aircraft-a380-842 (8K)** download link at the bottom of the page
